### ISSUE TITLE
chore: address a11y violations from various component docs (2)

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-react#7596]
- Tree view examples
-  Select examples
-  Navigation examples
-  Toggle group examples
-  Tab content examples
-  Page examples
-  Menu examples
-  Options menu examples
-  Masthead examples
